src/Atoms/Select/Select.jsx

```diff
@@ -1,14 +1,51 @@
-import React from 'react'
+import React, { Children } from 'react'
 import styled from 'styled-components'
-import NativeSelect from '@material-ui/core/NativeSelect'
+import { Select as MuiSelect } from '@material-ui/core'
+import { makeStyles, createStyles } from '@material-ui/core/styles'
 import { css } from '@styled-system/css'
 
+import Option from './Option'
+import InnerOption from './InnerOption'
 import Input from '../Input'
 import { useTranslation } from '../Trans'
 
-const BaseSelect = styled(NativeSelect)`
-  .MuiNativeSelect {
-    &-root { 
+const selectStyles = makeStyles(theme => createStyles({
+  icon: {
+    color:       theme.palette.label.main,
+    width:       '0.75em',
+    marginRight: 5,
+  },
+}))
+
+const inputStyles = makeStyles(theme => createStyles({
+  root: {
+    height:                  'auto !important', // FIXME: refacto input instead
+    borderBottomLeftRadius:  '0 !important', // FIXME: refacto input instead
+    borderBottomRightRadius: '0 !important', // FIXME: refacto input instead
+  },
+  input: {
+    padding: '6px 16px !important', // FIXME: refacto input instead

```

Comment:
Should we ask mesteche about this?

Reply:
He's the one who told me to add these `FIXME` (https://github.com/rgsystemes/lib-ui/commit/6fab4c42a94a0c721f71f7c9d6ee5b251b4b319d)